src/components/Callout/Callout.tsx

```diff
@@ -64,20 +66,52 @@ export class Callout extends React.Component<ICalloutProps, ICalloutState> {
   }
 
   public render() {
-    let { className, targetElement, isBeakVisible, beakStyle, children } = this.props;
+    let { className, targetElement, backgroundColor, closeButtonAriaLabel, isBeakVisible, hasCloseButton, beakStyle, children } = this.props;
     let { positions, slideDirectionalClassName } = this.state;
+
+    let backgroundClass;
+
+    switch (backgroundColor) {
+      case CalloutBackgroundColor.white:
+        backgroundClass = 'ms-Callout--white-bg';
+        break;
+      case CalloutBackgroundColor.blue:
+        backgroundClass = 'ms-Callout--blue-bg';
+        break;
+    }
+
     let content = (
       <div ref={ (host: HTMLDivElement) => this._hostElement = host } className={ 'ms-Callout-container' }>
         <div
-          className={ css('ms-Callout', className, slideDirectionalClassName ? `ms-u-${ slideDirectionalClassName }` : '') }
+          className={ css(
+            'ms-Callout',
+            className,
+            backgroundClass,
+            slideDirectionalClassName ? `ms-u-${ slideDirectionalClassName }` : '')
+          }
           style={ ((positions) ? positions.callout : OFF_SCREEN_POSITION) }
           ref={ (callout: HTMLDivElement) => this._calloutElement = callout }
           >
-          { isBeakVisible && targetElement ? (<div className={ beakStyle }  style={ ((positions) ? positions.beak : BEAK_ORIGIN_POSITION) } />) : (null) }
+          { isBeakVisible && targetElement ? (
+            <div
+              className={ beakStyle }
+              style={ ((positions) ? positions.beak : BEAK_ORIGIN_POSITION) }
+            />) : (null) }
           <Popup
             className='ms-Callout-main'
             onDismiss={ (ev:any) => this.dismiss() }
             shouldRestoreFocus={ true }>
+            { hasCloseButton && targetElement ? (
```

Comment:
You don't think there is any chance that another component would use that close  button? To me it feels more like part of the callout ui than a piece of content. I'd rather leave it somewhere it can be turned on with a single prop than having to recreate it again. I'm already having to recreate it after it was added to the dialog.